new_pass_logic.php

```diff
@@ -0,0 +1,62 @@
+<?php include "includes/header.php" ?>
+<?php include "includes/nav.php"; ?>
+<?php
+
```

Comment:
If you access this script via URL with an invalid token and try submitting the form like so: `/new_pass_logic.php?token=1234`, you see extra PHP warnings that you shouldn't.